Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM php:7.4
 
+RUN apt-get update -y && apt-get install -y zip
+
 RUN mkdir /twilio
 WORKDIR /twilio
+ENV PATH="vendor/bin:$PATH"
 
 COPY src src
+COPY tests tests
 COPY composer* ./
 
-RUN curl --silent --show-error https://getcomposer.org/installer | php
-RUN mv composer.phar /usr/local/bin/composer
-RUN composer install --no-dev
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
```

Comment:
Why this change instead of using curl?

Reply:
Figured it was more of a best practice to use the proper composer image rather than curl it